packages/os/configure-boot-mount.service

```
@@ -0,0 +1,16 @@
+[Unit]
+Description=Configure mount points for the active boot partition
+ConditionPathExists=!/proc/vmcore
+RefuseManualStart=true
+RefuseManualStop=true
+Wants=prepare-local.service
+After=prepare-local.service
```

We don't depend on `/local` any more.

I'd like to move this to the `release` package and align the unit name and description with `prepare-local.service`, i.e. `prepare-boot.service`. Maybe add a comment to indicate that finding the boot partition is not trivial and that's why we need the helper program.

```suggestion
[Unit]
Description=Prepare Boot Directory (/boot)
```